README.md

```diff
@@ -1,4 +1,7 @@
-# Confita
+[](https://user-images.githubusercontent.com/4570448/40786144-a8e8ce34-64ea-11e8-90b9-cb838cd13b88.png)
+
```

Comment:
There is a weird space before the logo, can you try removing these new lines ?

Reply:
I tried but i gave up. Show us your CSS skill !

Reply:
In that case, can you ask Mart for an image with the right size?